grpclb.go

```diff
@@ -568,6 +585,32 @@ func (b *balancer) Up(addr Address) func(error) {
 	}
 }
 
+// getNextConnectedAddr returns the next connected addr.
+// If no connected address is available, it returns nil.
+// b.next will be advanced to the correct value.
+// Caller must hold the lock.
+func (b *balancer) getNextConnectedAddr() *grpclbAddrInfo {
+	if len(b.addrs) > 0 && b.countConnected > 0 {
+		if b.next >= len(b.addrs) {
+			b.next = 0
+		}
+		next := b.next
+		for {
+			a := b.addrs[next]
+			next = (next + 1) % len(b.addrs)
+			if a.connected {
+				b.next = next
+				return a
+			}
+			if next == b.next {
+				// Has iterated all the possible address but none is connected.
```

Comment:
This should be impossible, because b.countConnected > 0, right?  Maybe comment "Should never happen"?